xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -1781,6 +1781,8 @@ void CVideoPlayer::ProcessRadioRDSData(CDemuxStream* pStream, DemuxPacket* pPack
 
 bool CVideoPlayer::GetCachingTimes(double& level, double& delay, double& offset)
 {
+  static bool cache_low_level = true;
```

Comment:
@FernetMenta : By hack, you mean using a static? So instead a member variable would be acceptable? I could also come up with a solution handled in CFileCache I think. Note sure whether it's worth doing it via ProcessInfo.h, that would be your call I guess?